test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/SchemaCompare/SchemaCompareServiceTests.cs

```diff
@@ -1371,6 +1371,16 @@ public async Task VerifySchemaCompareServiceCalls()
 
                 await SchemaCompareService.Instance.HandleSchemaCompareIncludeExcludeNodeRequest(excludeParams, publishRequestContext.Object);
 
+                // Include/Exclude all service call
```

Comment:
Given that the include/excludeAll logic lives at this layer, there should be tests that validate the actual input/output.  Should be easy to set up with some contrived dacpacs or sqlprojects.

Reply:
The requested test was added in the this commit: https://github.com/microsoft/sqltoolsservice/pull/2466/commits/b94508958df7f77ebfb242c144bc2b4eda95cefb